vscode/package.json

```diff
@@ -3,7 +3,7 @@
     "displayName": "Toit",
     "description": "Toit Programming Language Support",
     "publisher": "toit",
-    "version": "1.1.0",
+    "version": "1.0.0",
```

Comment:
I'm guessing we didn't publish with 1.1.0 yet?

Reply:
We didn't.